charts/lau-idam-backend/values.yaml

```diff
@@ -30,6 +30,6 @@ java:
         - idam-backend-POSTGRES-PASS
         - idam-backend-app-db-user
         - idam-backend-app-db-password
-        - idam-backend-flyway-password
+        - flyway-password
```

Comment:
This should be idam-flyway-password, it's picking up the value of the secret defined in line 67 of infrastructure/main.tf.